### PR TITLE
Galileo - Exposure

### PR DESCRIPTION
**Pull request by wwysoc2**
https://jira.opensciencedatacloud.org/browse/DICT-309

### CODE REVIEW

File: src/gdcdictionary/schemas/exposure.yaml [hunk 1]
```diff
@@ -641,6 +644,13 @@ properties:
       - Unknown
       - Not Reported
       - Not Allowed To Collect
+      - "1"
+      - "2"
+      - "3"
+      - "4"
+      - "5"
+      - "6"
+      - "7"
```
Comment (philtom-ctds) [inline src/gdcdictionary/schemas/exposure.yaml:653]: for my knowledge: as part of the sorting rules, are we moving deprecated enums to the end?

File: src/gdcdictionary/schemas/exposure.yaml [hunk 2]
```diff
@@ -294,6 +294,12 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  exposure_duration_hrs_per_day:
+    description: >-
+      The duration (in hours per day) that a person was exposed.
+    type: number
+    minimum: 0
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/exposure.yaml:301]: this isn't in the ticket, but is a max of 24 reasonable?

File: src/gdcdictionary/schemas/exposure.yaml [hunk 3]
```diff
@@ -294,6 +294,12 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  exposure_duration_hrs_per_day:
+    description: >-
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/exposure.yaml:298]: _terms has exposure_duration_hrs_per_day -- should this be a ref?

File: src/gdcdictionary/schemas/exposure.yaml (hunk 2 above)
Comment (wwysoc2) [reply src/gdcdictionary/schemas/exposure.yaml:302]: Good point, added to ticket and yaml.

File: src/gdcdictionary/schemas/exposure.yaml (hunk 3 above)
Comment (wwysoc2) [reply src/gdcdictionary/schemas/exposure.yaml]: Yes, changed to ref.

**Review by wteouchicago — APPROVED** [review]

**Review by sadhukd — APPROVED** [review]

**Review by stilesj-uchicago — APPROVED** [review]

**Pull request merged**